### PR TITLE
[2712] Update DfE Signin URL's

### PR DESCRIPTION
**Pull request by dankmitchell**
### Context
DfE Sign-in URL's

### Changes proposed in this pull request
Update "profile" and "search" URL's to use vanity URL.

### Guidance to review
🚢 

### Checklist
- [x] Make sure all information from the Trello card is in here
- [x] Attach to Trello card
- [x] Rebased master
- [x] Cleaned commit history
- [x] Tested by running locally

### CODE REVIEW

**Comment by dankmitchell** [comment]
[[S] Change URLs for DfE Sign-In Test Env](https://trello.com/c/WwOHewpB/2712-s-change-urls-for-dfe-sign-in-test-env)

**Comment by misaka** [comment]
It's weird that `development` and `pentest` provide both `profile` and `user_search_url` urls, but `qa and `test` only provide `profile`. What if we just removed all of them from `development`, `pentest`, `test` and `qa`, and relied on the setting to come from `settings.yml`?

File: config/settings/development.yml [hunk 1]
```diff
@@ -1,8 +1,6 @@
 dfe_signin:
-  issuer: https://signin-test-oidc-as.azurewebsites.net
-  profile: https://signin-test-pfl-as.azurewebsites.net
+  secret: please_change_me # Override with SETTINGS__DFE_SIGNIN__SECRET
```
Comment (misaka) [inline config/settings/development.yml]: As a general pattern, it's best if we use random values for default values that are used for authentication, for security. Even if this isn't a deployed env, there's still risks involved I believe.

```suggestion
  secret: <%= SecureRandom.base64 %> # Override with SETTINGS__DFE_SIGNIN__SECRET
```

Comment (dankmitchell) [reply config/settings/development.yml]: We don’t do that in any other settings file. Consistently wrong is better than inconsistent

Comment (dankmitchell) [reply config/settings/development.yml]: But yes I agree

Comment (misaka) [reply config/settings/development.yml]: We are moving things over slowly. For example https://github.com/DFE-Digital/manage-courses-frontend/blob/2712-update-dsi-urls/config/settings.yml#L18

We can do this as a bigger move if you prefer.

**Pull request merged**